skimage/restoration/_rolling_ball.py

```diff
@@ -38,20 +39,23 @@ def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=N
 
     Notes
     -----
-    For the pixel that has its background intensity estimated (without loss
-    of generality at ``center``) the rolling ball method centers ``kernel``
-    under it and raises the kernel until the surface touches the image umbra
-    at some ``pos=(y,x)``. The background intensity is then estimated
-    using the image intensity at that position (``image[pos]``) plus the
-    difference of ``kernel[center] - kernel[pos]``.
+    The algorithm is easy to grasp in 2D: Consider that each pixel value
+    defines a height, forming a 2D surface in 3D space. Then, a (3D) ball of
+    given radius (or a kernel, in the general case) is placed under this
+    surface and raised until it touches it. The background intensity is
```

Comment:
Ah, I see you have here what I added above. Fine to have it here, but I think we should avoid the term "translation", since it may make it sound like this is a linear algorithm. We need the user to understand that this is something rather different than convolution. Could also refer in the function description to the Notes for further detail.